Resources/Prototypes/Entities/Structures/Machines/bombs.yml

```diff
@@ -0,0 +1,148 @@
+- type: entity
+  abstract: true
+  parent: BaseStructure
+  id: BaseHardBomb
+  description: Just keep talking and nobody will explode.
+  name: hardbomb
+  components:
```

Comment:
these should have pointlights for Atmosphere. blu for training red for syndi i suppose